gammapy/irf/tests/test_effective_area.py

```diff
@@ -8,12 +8,36 @@
 from astropy.tests.helper import pytest
 from astropy.units import Quantity
 from numpy.testing import assert_allclose, assert_equal
+from collections import OrderedDict
 
 from ...datasets import gammapy_extra
 from ...irf import EffectiveAreaTable, abramowski_effective_area
 from ...utils.energy import EnergyBounds
 from ...utils.testing import requires_dependency, requires_data, data_manager
 
+@requires_data('gammapy-extra')
+def test_EffectiveArea2D(tmpdir):
+    # These are just some quick and dirty tests they should be removed later
```

Comment:
Can you put real tests now?
This is where I would start the review.

It's not clear to me what is already implemented here and what isn't.

One thing that would be nice to have is a repr like here that gives an overview of the axes and make the basic plots work again, to show that this works:
https://github.com/gammapy/gammapy/blob/master/gammapy/scripts/data_show.py#L92

Another example of what a repr with the axis info could look like: https://github.com/gammapy/gammapy/blob/master/gammapy/cube/core.py#L538

I.e. implement the things that are useful for playing around and debugging this.